admin/release.py

```diff
@@ -460,16 +482,110 @@ def upload_rpms(scratch_directory, target_bucket, version, build_server):
         )
 
 
-def publish_rpms_main(args, base_path, top_level):
+packages_template = (
```

Comment:
`html` doesn't claim Python 2.7 support on PyPI and the last release is from four years ago.  Also, now that I've looked at the examples, I'm a little skeptical of it (`escape=False`?  Terrible).